### ISSUE TITLE
Epub too big?

### ISSUE THREAD

**Issue by crazyyzarc** [xianmin/EveReader#13]
As it is, the action menu bar disappears when I want to read ebooks about larger than 20MB by selecting a chapter.

![2021-02-03 09_59_47-eve-reader](https://user-images.githubusercontent.com/4954535/106722972-e273aa00-6606-11eb-9f2d-a89e500e6c71.png)

**Comment by xianmin** [comment]
This is very strange. Do other big epub files do the same thing?

**Comment by crazyyzarc** [comment]
> 
> 
> This is very strange. Do other big epub files do the same thing?

yes every epub  >= ~20mb, but I have only 2-3 epubs in these size

**EDIT**: Okay, I have tested more books and can now say that only my first three books cause this behavior

**Comment by xianmin** [comment]
Can you send me a file and I test it? 
My email: xianmin12@qq.com 
Thank you!
